bundles/org.openhab.binding.emotiva/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="emotiva" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Emotiva Binding</name>
+	<description>This is the binding for devices from the Emotiva Audio Corporation.</description>
+	<connection>local</connection>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>ip</service-type>
+			<discovery-parameters>
+				<discovery-parameter>
+					<name>type</name>
+					<value>ipBroadcast</value>
+				</discovery-parameter>
+				<discovery-parameter>
+					<name>destPort</name>
+					<value>7001</value>
+				</discovery-parameter>
+				<discovery-parameter>
+					<name>timeoutMs</name>
+					<value>1000</value>
+				</discovery-parameter>
+			</discovery-parameters>
+		</discovery-method>
```

Comment:
@espenaf - how did you test this? I just noticed this started appearing in my logs:

```
2024-06-18 21:54:43.493 [WARN ] [fig.discovery.addon.ip.IpAddonFinder] - binding-emotiva: discovery-parameter 'request' or 'requestPlain' required
```

This discovery method seems incomplete as it also lacks `match-properties`. For a full example, see here:

https://github.com/openhab/openhab-addons/blob/54ba2c732b0f12f2df5d6de72723fcd018de73f9/bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/addon/addon.xml#L11-L39

Reply:
I did test the discovery, so that is working. Does this missing request or requestPlain parameter block the actual discovery in any way, or is it just helpful metadata?

The reason I did not add any request property is that the discovery needs this EmotivaPing XML structure, so it did not thing it would fit for this values. I see now that the ipcamera binding is having the same "problem", and it has actually added a XML escaped value in the requestPlain, so I could probably do the same. Something like:

```
				<discovery-parameter>
					<name>requestPlain</name>
					<value>&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;&lt;emotivaPing protocol=&quot;2.0&quot;/&gt;</value>
				</discovery-parameter>
```

Reply:
Just to be sure, when you mention you tested the discovery, do you mean the suggested binding feature:

![image](https://github.com/openhab/openhab-addons/assets/19519842/c652fc2c-936a-4844-a643-43ebd88b058d)

or the discovery within the binding?

The logged message means there's an error in the defined `discovery-method`, and it will not work.

Reply:
Ohh, the suggested binding feature was news to me. Then the whole discovery-methods made a bit more sense. After some tweaking of the values, for instance changing ipBroadcast to ipMulticast and adding requestPlain and adding match-properties with reponse=.*, it has stopped giving warnings, but it is not actually discovering anything yet. 

How do I trigger the discovery to start? I got DEBUG setup for org.openhab.core.config.discovery.addon so I am seeing what bundles are trying to do discovery, but my binding is not being triggered. I am running this on openhab 4.1.3 with the newly compiled code added as a jar file.

Reply:
Please create a PR with your changes and we can continue discussion from there.

Reply:
Opened https://github.com/openhab/openhab-addons/pull/16891